src/components/window-shade.sass

```diff
@@ -30,5 +31,43 @@
   padding-bottom: 28px;
 
 .windowShadeContentHide
-  height: 0px
+  height: 3px
   overflow: hidden
+  border-top: #878706 solid 3px
+
+.content
+  font-family: Arial, sans-serif
+  font-size: 16pt
+  font-weight: regular
```

Comment:
This value is incorrect and there are a couple of semi-colons in this file, and my IDE complains about that. 
If you don't like SASS, we can switch to SCSS. I only used SASS as that's what the starter project used, but I'm not a big fan of it. I keep adding semi-colons all the time too. :wink:

Reply:
Since .scss seems to be the more current format I think it would make sense for the starter-project to use it instead. I think Doug may have run into an issue getting .scss to compile in CLUE at some point but I don't remember the details. In any case, it's probably worth fresh consideration.

Reply:
I'm not sure what the error is, but I'll chase that down when we do the final styling. In the mean time, I've removed all the semi-colons and I'll check the other SASS files for semi-colons. As per our discussion, earlier this morning, we'll stick to SASS and maybe discuss a conversion to SCSS in the next iteration. I personally (DL) have no opinion, but I am very happy to move to SCSS.